### DISCUSSION TITLE
Consider adding Qgrid as a general form of filter/selector for plotting

### DISCUSSION THREAD

**Discussion by taoluo** [datapane/datapane#35] [Ideas]
I am not sure if it is feasible to integrate, but this form of per-column filter is very flexible. 
https://github.com/quantopian/qgrid

**Comment by taoluo** [comment]
A demo from https://github.com/quantopian/qgrid#events-api

![image](https://user-images.githubusercontent.com/12562760/97807786-e5749c80-1c9d-11eb-9f13-f29f6b8269e2.png)

**Comment by mands** [comment]
Hi there,

This is a great feature request - I think supporting QGrid itself may be a  bit tricky, as we don't have Python running when you're viewing a report in the browser, but it's certainly possible that we can add similar functionality via the web using JS.

We're planning on overhauling our table component in the new few weeks so hopefully we'll have something more to share shortly.
